backend/common/src/test/java/ai/verta/modeldb/common/db/migration/MigrationTest.java

```diff
@@ -11,14 +10,16 @@ class MigrationTest {
   void ordering() {
```

Comment:
This is a great candidate for a property test.  
1. Build a generator that produces the set of migrations. 
2. Write a clean-room brute force method that processes the set and returns the known correct result.
3. Execute the clean-room method and the method-under-test and compare the results

Reply:
Heh. I _just_ commented in slack that I wanted to write one.

Reply:
I do worry a little bit about the "clean room" sort also being tested by a property test like this. What if they were both wrong?

Reply:
Better might be: generate a random set of valid Migration instances, put them in a set, then validate the order desired is strictly enforced (that is... monitonically increasing numbers with up always before down).